### ISSUE TITLE
NullPointerException is thrown when variable declaration with initial value is inputed

### ISSUE THREAD

**Issue by syohex** [konoha-project/libzen#12]
I got following error.

```
% java -jar libzen.jar
LibZen0.1 (Reference Implementation of D-Script) on Java JVM-1.7.0_25
Copyright (c) 2013-2014, Konoha project authors
Accept: zen0.1
Produce: zen0.1
>>> var c = 10;

java.lang.NullPointerException
    at zen.lang.ZenVarInfo.<init>(ZenVarInfo.java:47)
    at zen.lang.ZenTypeChecker.SetVarInfo(ZenTypeChecker.java:409)
    at zen.lang.ZenDynamicTypeChecker.VisitVarDeclNode(ZenDynamicTypeChecker.java:332)
    at zen.ast.ZenVarDeclNode.Accept(ZenVarDeclNode.java:64)
    at zen.lang.ZenTypeChecker.TypeCheck(ZenTypeChecker.java:295)
    at zen.parser.ZenNameSpace.TypeCheck(ZenNameSpace.java:582)
    at zen.parser.ZenNameSpace.Eval(ZenNameSpace.java:593)
    at zen.main.ZenMain.ExecCommand(ZenMain.java:213)
    at zen.main.ZenMain.main(ZenMain.java:266)
    at <TopLevel>.TopLevelEval(stdin:1)
```

**Comment by imasahiro** [comment]
I cannot reproduce this error. I think this issue is related to  issue #11 and commit 1581139 fix this error.

**Comment by syohex** [comment]
My Java environment is 

```
% java -version 
java version "1.7.0_25"
OpenJDK Runtime Environment (IcedTea 2.3.12) (7u25-2.3.12-4ubuntu3)
OpenJDK 64-Bit Server VM (build 23.7-b01, mixed mode)
```

**Comment by imasahiro** [comment]
@syohex  Thank you for your infomation. I reproduced NullPointerException but stack trace seems different.

```
LibZen0.1 (Reference Implementation of D-Script) on Java JVM-1.7.0_45
Copyright (c) 2013-2014, Konoha project authors
Accept: zen0.1
Produce: zen0.1
>>> var x = 10;

java.lang.NullPointerException
        at zen.lang.ZenTypeChecker.NewVarType(ZenTypeChecker.java:513)
        at zen.lang.ZenDynamicTypeChecker.VisitVarDeclNode(ZenDynamicTypeChecker.java:323)
        at zen.ast.ZenVarDeclNode.Accept(ZenVarDeclNode.java:69)
        at zen.lang.ZenTypeChecker.TypeCheck(ZenTypeChecker.java:398)
        at zen.parser.ZenNameSpace.TypeCheck(ZenNameSpace.java:582)
        at zen.parser.ZenNameSpace.Eval(ZenNameSpace.java:593)
        at zen.main.ZenMain.ExecCommand(ZenMain.java:213)
        at zen.main.ZenMain.main(ZenMain.java:266)
        at <TopLevel>.TopLevelEval(stdin:1)
```

Maybe problem is in typechecker. I will check it.
thanks

**Comment by syohex** [comment]
This issues is fixed at 731ecb5dcf70f9f23967cdb7a23dbef27c8de8a4 and a182dc1e1d2d3c9f28e3762900525978e25f297c. I close.
